scheduler/cups-driverd.cxx

```diff
@@ -113,11 +115,17 @@ typedef union				/**** TAR record format ****/
   }	header;
 } tar_rec_t;
 
+struct args {
+    cups_file_t *fp;
+    char *buf;
+    size_t buflen;
+};
 
 /*
  * Globals...
  */
 
+static char* retval=NULL;
```

Comment:
@pranjanpr Global variables must be capitalized and have a comment what they are meant for. And IMO ```retval``` is too generic name.